.github/workflows/test_frameworks.yml

```
@@ -94,6 +94,36 @@ jobs:
         # django.tests.requests module interferes with requests library patching in the tracer -> disable requests patch
         run: DD_TRACE_REQUESTS_ENABLED=0 ddtrace-run tests/runtests.py
 
+  graphne-testsuite-3_0:
+    runs-on: ubuntu-latest
+    env:
+      DD_PROFILING_ENABLED: true
+      DD_TESTING_RAISE: true
+    defaults:
+      run:
+        working-directory: graphene
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v3
+        with:
+          repository: graphql-python/graphene
+          ref: v3.0.0
+          path: graphene
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.8"
```

```suggestion
          python-version: "3.10"
```

unless there's a reason to not use the latest?